events/spells.js

```diff
@@ -0,0 +1,23 @@
+function activateTemporaryPocket(context) {
+  const duration = 120 * 1000;
+  const playerId = context.id();
+  const { id, players } = context.getGameState();
+  const game = context.getGameState();
+
+  players[playerId].properties.temporaryPockedLocked = false;
+  const newGameState = { ...game };
+  context.updateGameState(newGameState, id);
```

Comment:
Since we're changing the state of a game where the player is already connected the second argument here isn't necessary, so `context.updateGameState(newGameState)` is enough 👍🏻